docs/src/cookbooks/dataview.md

```diff
@@ -177,3 +178,35 @@ As indicated in the comment, abstract methods must still be implemented.
 This is the same that happens when one writes `new Bundle {}`,
 the curly braces create a new concrete subclass; however, because `Bundle` has no abstract methods,
 the contents of the body can be empty.
+
+### How can I use `.viewAs` instead of `.viewAsSupertype(type)`?
+
+While `viewAsSupertype` is helpful for one-off casts, the need to provide a type template object
+each time can be onerous.
+Because of the subtyping relationship, you can create a `PartialDataView` from a Bundle type to
+a parent type by just providing the function to construct an instance of the parent type from
```

Comment:
should you mention specifically the `PartialDataView.supertype` function here?

Reply:
Yeah that makes sense, rephrased slightly.